### ISSUE TITLE
Include example presets to make use of the extension easier.

### ISSUE THREAD

**Issue by SharkWipf** [RomainLK/ytc-filter#32]
I've been using this extension for a while and it's been really useful, but currently the barrier to entry, at least for its main use case in vtuber youtube, is fairly high.
Having stock presets/profiles available for i.e. the common translation filters would make getting started a lot easier.
Having sane "default" filters available could also be helpful to live translators, who would have a clear indication of what formats they could stick to.  

Personally I use [/u/WhisperingForestWind](https://www.reddit.com/r/Hololive/comments/ii5naq/ytcfilter_an_extension_to_help_you_filter_the/g362ycb/)'s regex for English translation, but there may be better defaults available

**Comment by RomainLK** [comment]
I know that it's not the best extension in term of UX and ease of use and some future features are asking for even more complexity while the interface is still stuck in the small corner of the Youtube chat.

For now I'm trying to have people read the wiki which offers a guide. There is a regex cookbook there and considering that there is now an import/export filter function, I could add a new section for them.

For integrating presets, yes this could be useful, perhaps in some kind of wizard. Or they could be profiles available at first install, and I rename profile in preset which they kind of are actually, and I show more prominently.

**Comment by SharkWipf** [comment]
As someone who discovered this extension on Reddit, I must admit I had no idea there even was a wiki.  
That said, with this change implemented I don't feel like it _needs_ a wiki for most users' usage either, as for its main intended use case (live translation), the requirements/filters should be the same for almost all users (maybe different language filters), and more specific filters generally only add to that.  
I believe having "sane defaults" would pretty much make most current UX shortcomings irrelevant.

**Comment by RomainLK** [comment]
The wiki is present within the first time message you get when installing/updating and at the bottom of the options. But I already figured that it's easily overlooked 😢 

While I intended this extension to be used for translation, there are other use case I have seen which could get their own preset. I suppose that when I'm ready to implement the feature, I will ask around for people to share their filter with me.

I have opened another ticket which is an experiment for me to break away from the small screen space I have to work with.  I will have this as a requirement for future features. Presets will then be an improved profile, as profiles today lack descriptions and preview of their content.
